src/Scanner/MethodScanner.php

```diff
@@ -452,7 +452,7 @@ protected function scan()
                 $tokenType    = null;
                 $tokenContent = $token;
                 $tokenLine    = $tokenLine + substr_count(
-                    $lastTokenArray[1],
+                    $lastTokenArray[1] ?? null,
```

Comment:
I guess it works, but I would prefer another fix: we shouldn't call `substr_count` if the first argument is null. This is not a problem now, but when we add strict types it becomes an issue, see: https://www.php.net/substr_count

Reply:
Yes, this is a quick trivial fix (keeping same behavior)
But indeed, there is a logic error is this part of the code.